pkgs/development/libraries/libetpan/default.nix

```diff
@@ -1,31 +1,29 @@
 { stdenv, fetchFromGitHub
-, autoconf
-, automake
-, libtool
-, openssl
+, autoconf, automake, libtool, openssl, pkg-config
 }:
 
 stdenv.mkDerivation rec {
   pname = "libetpan";
-  version = "1.9.3";
+  version = "1.9.4";
 
   src = fetchFromGitHub {
     owner = "dinhviethoa";
     repo = "libetpan";
     rev = version;
-    sha256 = "19g4qskg71jv7sxfxsdkjmrxk9mk5kf9b6fhw06g6wvm3205n95f";
+    sha256 = "0g7an003simfdn7ihg9yjv7hl2czsmjsndjrp39i7cad8icixscn";
   };
 
-  nativeBuildInputs = [ libtool autoconf automake ];
+  nativeBuildInputs = [ autoconf automake libtool pkg-config ];
```

Comment:
Did you check if

```
nativeBuildInputs = [ autoreconfHook pkg-config ];
```

works, removing the `configureScript` line below?

Reply:
Thanks for this suggestion. I tried this, but it fails due to missing files which should (at least I think so) be generated by Autoconf.

Reply:
Ack, thanks for trying!

Reply:
I'm not exactly an Autoconf expert, to put it like that. If anyone wants to take care of it, I'd be happy.